packages/headless/src/commerce.index.ts

```diff
@@ -33,6 +33,9 @@ export * from './features/commerce/query-suggest/query-suggest-actions-loader';
 export * from './features/configuration/configuration-actions-loader';
 export * from './features/commerce/query/query-actions-loader';
 
+// Selectors
+export {responseIdSelectorFromEngine} from './features/commerce/product-listing/product-listing-selectors';
```

Comment:
We should discuss how to organize those selector exports.

Just exporting them all at root like this will balloon up to hundreds of exported symbols very quickly, which is not good for implementers discoverability when your are in an IDE and see a huge list of available functions from `@coveo/headless/commerce`.

How about namespacing those export for now ?